Article/Article.js

```diff
@@ -1,26 +1,87 @@
 // Start at `let articles`
 
 class Article {
-  constructor(element) {
+  constructor(article) {
     // assign this.element to the passed in article element
-    this.element;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.article = article;
+    // create a reference to the ".expandButton" class.
+    this.expandButton = this.article.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = 'Click to expand';
     // Set a click handler on the expandButton reference (or article element), calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
+    // Close Article
+    this.article.addEventListener('click', e => this.closeArticle(e));
+  }
+
+  closeArticle(e) {
+    if (e.target.classList.contains('close')) {
+      TweenMax.to(this.article, 1, {
+        x: 2000,
+        ease: Bounce.easeOut
+      });
+      TweenMax.to(this.article, 0.5, {
+        height: 0,
+        delay: 1
+      });
+      setTimeout(() => (this.article.style.display = 'none'), 2000);
```

Comment:
I think tweenMax take a call back function, so you could have this go there instead, which would be a bit safer, since now you're relying on two async events to line up propertly, but that's not guaranteed.

Awesome approach though!